project/client/cmd/root.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/m/v2/client/cmd/core"
+	"github.com/spf13/cobra"
+)
+
+var RootCmd = &cobra.Command{
```

Comment:
No need for capital "R"... You are not using this variable anywhere else right?

Capital exports the variable out of this package.... so we use capital only when we need to access it from outside